akka-docs/src/main/paradox/typed/fault-tolerance.md

```diff
@@ -116,9 +116,9 @@ Java
 It is possible to override this so that child actors are not influenced when the parent actor is restarted.
 The restarted parent instance will then have the same children as before the failure.
 
-If child actors are created from `setup` like in the previous example and they should remain intact (not stopped)
-when parent is restarted the `supervise` should be placed inside the `setup` and using
-@scala[`SupervisorStrategy.restart.withStopChildren(false)`]@java[`SupervisorStrategy.restart().withStopChildren(false)`]
+If child actors are created from @apidoc[setup](typed.*.Behaviors$) {scala="#setup[T](factory:akka.actor.typed.scaladsl.ActorContext[T]=%3Eakka.actor.typed.Behavior[T]):akka.actor.typed.Behavior[T]" java="#setup(akka.japi.function.Function)"} like in the previous example and they should remain intact (not stopped)
+when parent is restarted the @apidoc[supervise](typed.*.Behaviors$) {scala="#supervise[T](wrapped:akka.actor.typed.Behavior[T]):akka.actor.typed.scaladsl.Behaviors.Supervise[T]" java="#supervise(akka.actor.typed.Behavior)"} should be placed inside the @apidoc[setup](typed.*.Behaviors$) {scala="#setup[T](factory:akka.actor.typed.scaladsl.ActorContext[T]=%3Eakka.actor.typed.Behavior[T]):akka.actor.typed.Behavior[T]" java="#setup(akka.japi.function.Function)"} and using
+@scala[@scaladoc[SupervisorStrategy.restart.withStopChildren(false)](akka.actor.typed.RestartSupervisorStrategy#withStopChildren(enabled:Boolean):akka.actor.typed.RestartSupervisorStrategy)]@java[@javadoc[SupervisorStrategy.restart().withStopChildren(false)](akka.actor.typed.RestartSupervisorStrategy#withStopChildren(boolean))]
```

Comment:
I don't like the long line `SupervisorStrategy.restart.withStopChildren(false)`, but I need some different texts:
scala - `SupervisorStrategy.restart.withStopChildren(false)` (`restart` and then POINT)
java - `SupervisorStrategy.restart.withStopChildren(false)` (`restart` and then PARENTHESES).

I can use apidoc and the same text. Should I do it?

Reply:
I don't have a preference.